datafusion/sql/tests/sql_integration.rs

```diff
@@ -54,15 +54,6 @@ use sqlparser::dialect::{Dialect, GenericDialect, HiveDialect, MySqlDialect};
 mod cases;
 mod common;
 
-#[test]
-fn test_schema_support() {
```

Comment:
I move all the test including `select *` to slt files. Add explain statement for them